web/themes/contrib/civictheme/src/CivicthemeConfigManager.php

```
@@ -49,7 +51,8 @@ public static function create(ContainerInterface $container): self {
       $container->get('config.factory'),
       $container->get('theme.manager'),
       $container->get('extension.list.theme'),
-      $container->get('class_resolver')->getInstanceFromDefinition(CivicthemeConfigImporter::class)
+      $container->get('class_resolver')->getInstanceFromDefinition(CivicthemeConfigImporter::class),
```

PHPStan (2.1.39)

[error] Call to an undefined method object::getInstanceFromDefinition().


(method.notFound)